### ISSUE TITLE
UI font-size / elements too small

### ISSUE THREAD

**Issue by lab-at-nohl** [tine-groupware/tine#125]
It's been bothering me for years. The content of the UI is unreadable small, especially during a work day when your eyes become tired, Tine renders unusable. I don't know how you manage it. I personally zoom in a lot; every browser supports that,(I know the key shortcut and do it automatically. Now and then, however, zoomed elements break the UI and I "compensate" this by manual zoom (but that's only me).  

Today I have talked to someone, introducing Tine. Because it wasn't my device, everything was in default - way too small. I realized then how small it really is! The showcase was over, though ("looks like 90s..."). 

Just to clear the dimension: 

- Elements become distinguishable with a **minimum zoom of 133 %** the earliest. On a standard notebook / desktop display without zoom, text is literally not readable. Meaning you are guessing the letters not reading due to "lost" pixels. 
- SVG images start to appear clean only with zoom, the displays I tested it needs **at least 150 %**. 
- Standard font-sizes in the panels, including specially designed contents like calendar sheets, also render clean at 150 % the earliest. 
- For "working content" like the Text of emails I would prefer even 200% zoom. This may not be suitable as a global setting. However, to really work with text without eye tiredness it would be preferable. 

I am not a designer. But you can read all over the web that typically **regular text is not smaller than 16px** (Tine: 11px standard elements), 11px is tolerated only for additional illustration like Button description or "badges". I also realize that this changed over time as displays got higher resolution. However, today I would call it a bug (I got it mirrored today by someone "neutral")

As a quick relieve: I do not see any harm in adjusting everything by 133 %. Do you? 

Because of Tine I had look into OnlyOffice - they do it very right UI wise...

**Comment by ignoreigor** [comment]
I would append that the font-aliasing  is bad when using the dark theme. Maybe it's a subjective impression, but compare yourself:

![Image](https://github.com/user-attachments/assets/97bf86d8-c7c9-45df-b8c0-15878f6ea200)
![Image](https://github.com/user-attachments/assets/cd24b17f-638b-4bd0-a6e6-dc94a012ab89)

![Image](https://github.com/user-attachments/assets/a4f9334b-3642-45ca-a016-17e7c45d95f8)
![Image](https://github.com/user-attachments/assets/0b3b0323-5062-4a13-9c68-125ee369c016)

**Comment by lab-at-nohl** [comment]
> I would append that the font-aliasing is bad when using the dark theme. Maybe it's a subjective impression, but compare yourself:

Have you tried using the browser's zoom feature? I also like the dark theme. On my Linux notebook/desktop as well as on Windows server, the font is crystal clear when you zoom to at least 150%.

**Comment by ignoreigor** [comment]
I tried it both in Firefox and Edge  on Windows with FullHD-Display. I believe I tried it in FF on Linux as well and got this blurry and unregular effects. But well, it is getting better with zooming. 125 percent is better, with scale of 150% it is fine. 
Yes, the font-size in tine is very small, but I like the overview of the monthly calendar. With zooming, you are getting "x more" very fast. But of course bigger fonts is more relaxing to read, for me scaling up to 120% would be sufficient (at the moment).

**Comment by pschuele** [comment]
thanks for the feedback and suggestions - we have  an internal project "tine accessibility", where I linked this issue. we'll give you updates on the progress.

**Comment by pschuele** [comment]
got some feedback from Conny:
- only changing the size/font in the css is tricky - the layout needs to be adjusted
-  it would be great to have some help, maybe someone can experiment with the css and send screenshots?
-  the "big plan" is to use/switch to bootstrap-css
-  we also need to remove dark-mode invert

**Comment by lab-at-nohl** [comment]
I tried to understand how css works with Tine - but failed. It seems to me that this ancient ExtJs 3.1.1 does hard-code font-size into each element. In older days there was theming (unfortunately it still exists in library/ExtJS...), but also back then, I guess, it was solely about changing colors. Nevertheless you managed to overwrite the appearance of all elements obviously. 

I am really lost here.

What you can do for an impression: If you use Firefox, in the old menu (Alt key) View/Zoom there's an option "Zoom Text only". I guess Conny is right, the layout is affected; However, it would fit mostly for an text-only zoom of 120 % (including complex things like Calendar). But for readability a minimum of 133 % would be nice.

**Comment by lab-at-nohl** [comment]
I hope something is happening in the background. If ExtJs is the problem, maybe ai can help to resize every element? (I will listen your podcast regarding ai next). Also I saw commits for the ui lately. However, this issue needs to be resolved imho…

**Comment by pschuele** [comment]
you are right, this is an important issue and should not be forgotten. assigning this to @corneliusweiss because he is coordinating the ui glow-up.
